src/boards/boardacorn.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace GOL\Boards;
+
+use GOL\Rule;
+
+/**
+ * Represents a Game of Life world filled with an acorn in the center.
+ *
+ * Use grid() to retrieve the current grid and setGrid() to apply all changes.

```

Comment:
Why is this important for the BoardAcorn class? (By the way I would prefer `AcornBoard` as name, but it is up to you to decide on that)